### DISCUSSION TITLE
MobaXterm_Portable_v23.0_zh-CN-beta.1

### DISCUSSION THREAD

**Discussion by RipplePiam** [RipplePiam/MobaXterm-Chinese-Simplified#32] [General]
**🎉🎉🎉 MobaXterm Portable v23.0 (2023-01-27) 汉化版发布啦 !**

**更新说明：**

- 本次更新可解决 #30 #29 #27 #26 #12 
- 全面支持 Windows 11
- 主观体验上各种图标会更高清

**软件更新日志（来自官方网站）：**
**(1) 版本 23.0 (2023-01-27)**

> 改进：添加了基于 Microsoft Edge WebView 的新 Web 浏览器引擎
> 改进：添加了新的 RDP“缩放”功能，允许您为远程 RDP 会话指定缩放级别
> 改进：添加了一个新的 RDP 设置“身份验证级别”，这在您使用 RDP over SSL 时很有用
> 改进：添加了一个新的 RDP 设置“颜色深度”，以便在客户端指定所需的颜色深度
> 改进：添加了一个新的 RDP 设置“重定向智能卡”，允许在服务器端使用本地智能卡，甚至用于身份验证
> 改进：以更方便的方式安排 RDP 高级设置
> 改进：在基于 Edge/WebView 的浏览器会话中添加了代理支持
> 改进：使用嵌入式引擎改进了 VNC 会话的整体速度
> 改进：增加了端口字段的宽度，以便更好地读取大于 9999 的端口号
> 改进：在专业定制程序中添加了一个新设置以禁用远程监控功能
> 改进：现在可以从定制程序中选择 64 位终端
> 改进：当 Slash 目录设置为“Program Files”文件夹时，插件现在在安装后解压
> 改进：您现在可以在 MobaXterm Customizer 中包含 64 位插件
> 错误修复：修复了设置跳转主机并将 Slash 目录设置为“Program Files”文件夹时 SSH 命令行客户端的问题
> 错误修复：修复了设置新的嵌入式引擎和 SSH 网关时 VNC 会话可能发生的崩溃
> 错误修复：将旧的自定义文件导入定制程序时，本地终端设置未正确恢复
> 错误修复：修复了在定制器中将斜线目录设置为“与应用程序相同”时 OpenSSH 守护进程的启动
> 漏洞修复：修复了 SSH 密钥会话字段中显示的 Unicode 字符
> 漏洞修复：修复了启动文件夹会话字段中显示的 Unicode 字符
> 漏洞修复：修复了固定 Home 和固定 Slash 文件夹字段中显示的 Unicode 字符

**(2) 版本 22.3 (2022-12-12)**

> 改进：将每个实例的最大选项卡数量从 36 增加到 100
> 改进：您现在可以使用 Ctrl+V 或 Shift+Insert 将一些文件粘贴到 SSH 浏览器
> 改进：您现在可以使用 Ctrl+V 或 Shift+Insert 将一些文件粘贴到 FTP、SFTP 或 S3 会话
> 改进：3 个 INI 配置设置 TermReconnectKey、TermSaveKey 和 TermExitKey 现在可以设置为特殊键（例如“Return”、“F4”、“Backspace”等）
> 改进：提高了嵌入式 VNC 查看器的速度
> 改进：现在使用服务器地址和端口识别 VNC 密码
> 改进：VNC 密码以正确的服务器名称保存，即使在使用 SSH 网关或跳转主机时也是如此
> 改进：向嵌入式 VNC 查看器添加了几种新的身份验证方法（Plain、SSL、TLS、X509Vnc 和 X509Plain）
> 改进：在嵌入式 VNC 查看器中添加了 SSL 隧道连接功能
> 改进：在嵌入式 VNC 查看器中添加了带有登录名和密码的 Unix 身份验证支持
> 改进：在嵌入式 VNC 查看器中添加了 SOCKS4、SOCKS5 和 HTTP 代理支持
> 改进：您现在可以选择在当前选项卡旁边放置一个重复选项卡（在“杂项”全局设置部分中）
> 改进：您现在可以选择将每个新选项卡放置在当前选项卡旁边
> 改进：添加了一个新的“OpenVMS2”终端类型，增加了应用程序键盘功能和 PF1-PF4 键
> 改进：使用“-exitwhendone”命令行参数时，终端不提示重新连接
> 改进：增加了非活动选项卡的颜色饱和度
> 改进：当 MobaXterm 在隐藏模式下启动时，密码提示不再打印到隐藏终端
> 漏洞修复：修复了将 AWS SSM 插件用作本地代理时与 AWS 的 SSH 会话问题
> 错误修复：在拆分模式下，当第一个终端标题更新时，主窗口标题现在会正确更新

**(3) 版本 22.2 (2022-11-15)**

> 改进：更新本地终端环境到最新版本
> 改进：更新了通过“apt”下载的软件包
> 改进：提高了“apt”和“MobApt”包管理器的速度
> 改进：更好地检测 Windows 控制台应用程序中的特殊 Unicode 字符
> 改进：为 Git-Bash 会话将“MSYSTEM”变量设置为“MINGW64”
> 改进：默认将首选 PATH 设置为 /usr/bin 而不是 /bin
> 改进：统一的配置文件脚本，以便在所有本地 shell 中具有相似的功能
> 改进：删除了不再需要的 MobaSendMsg 二进制文件
> 改进：删除不再需要更快启动的“mkpasswd”和“mkgroup”
> 改进：您现在可以精确选择本地终端外壳和分发
> 改进：您现在可以为 MobaXterm 本地终端选择外部分发
> 改进：在本地终端环境和“Shell”会话中添加了 ZSH 支持
> 改进：处理多个会话（超过 1000 个）时的一些速度改进
> 改进：远程监控状态栏大于窗口时可以滚动
> 改进：远程监控状态栏现在可以选择显示NFS挂载分区
> 改进：默认情况下为 RDP 会话禁用“重定向驱动器”、“重定向打印机”和“重定向端口”
> 改进：为终端重新连接热键、TermReconnectKey、TermSaveKey 和 TermExitKey 添加了 3 个 INI 配置设置
> 改进：在 Professional Customizer 程序中添加了默认安全预设
> 改进：添加了一个新设置以禁用 Professional Customizer 程序中的“MobApt”包管理器
> 改进：添加了一个新设置以禁用 Professional Customizer 程序中的“RDP 重新连接”功能
> 改进：在便携式模式下，代理键路径自动使用“CurrentDrive”或“CurrentFolder”变量
> 改进：您现在可以为每个 SSH 会话选择 SSH 协议版本
> 改进：您现在可以对每个 SSH 会话的密钥交换算法列表进行排序
> 改进：您现在可以对每个 SSH 会话的主机密钥算法列表进行排序
> 改进：您现在可以对每个 SSH 会话的密码算法列表进行排序
> 改进：您现在可以选择阻止 MobaXterm 向服务器公布它已知的主机密钥算法
> 改进：您现在可以为每个 SSH 会话选择启用或禁用代理身份验证和代理转发
> 改进：改进了对具有不同 DPI 设置的多显示器的支持
> 改进：改进了在 MobaXterm 打开时更改 Windows 分辨率的支持
> 改进：防止 MobaXterm 在辅助 4K 显示器上启动时变得模糊
> 改进：改进了 MobaXterm 和 MobaTextEditor 图标的清晰度
> 改进：防止 RDP 会话在辅助监视器上变得模糊
> 改进：提高了嵌入式资源解压的速度
> 错误修复：删除了在未经身份验证的情况下使用代理连接启动隧道时的密码提示
> 错误修复：当 Windows DPI 设置为 125% 时，“粘贴确认”对话框中的下划线未正确呈现
> 错误修复：在某些特定情况下，使用双击展开会话文件夹可能会导致启动第一个会话
> 错误修复：修复了 SSH 连接密码未存储到 MobaXterm 时 SCP 浏览器中的一些下载问题
> 错误修复：X11 转发在特定情况下在与代理的 SSH 会话中失败
> 错误修复：修复了 SSH 会话的连接问题，当在同一会话中配置了 SOCKS 代理和跳转主机时
> Bugfix：在SFTP会话认证提示中点击“取消”后，连接被正确中止
> 错误修复：启动 SFTP 会话时正确检查远程服务器主机密钥（修复 CVE-2022-38336，由奥地利 MilCERT 的 Manfred Kaiser 报告）
> 错误修复：当 SFTP 会话中止时，“已取消”密码不再发送到服务器（修复 CVE-2022-38337，由奥地利 MilCERT 的 Manfred Kaiser 报告）

<hr /><em>This discussion was created from the release <a href='https://github.com/RipplePiam/MobaXterm-Chinese-Simplified/releases/tag/v0.23.109'>MobaXterm_Portable_v23.0_zh-CN-beta.1</a>.</em>

**Comment by baibai1989** [comment]
很好用，请问一下，mobatexteditor有汉化么，要设置的还是还没汉化的

**Reply by RipplePiam** [reply]
只汉化软件本身，内置工具不作汉化
